ssl_exporter.go

```diff
@@ -375,12 +375,13 @@ func main() {
 	log.Infoln("Starting "+namespace+"_exporter", version.Info())
 	log.Infoln("Build context", version.BuildContext())
 
+	//lint:ignore SA1019 because
 	http.Handle(*metricsPath, prometheus.Handler())
```

Comment:
Rather than ignoring the check, could we not just switch to `promhttp.Handler()` as suggested?

Reply:
this ignore is only been moved from https://github.com/ribbybibby/ssl_exporter/blob/606f4f6032356292fea9b0f09c787a5be00511de/Makefile#L18 (different linter). didn't dive into doing the actual switch

Reply:
Aha, okay. I see. I can look at that separately.